### ISSUE TITLE
Universal Viewer .pdf Support

### ISSUE THREAD

**Issue by sourslaw** [UMNLibraries/cdm-blacklightify#130]
## Descriptive Summary
Double check Universal Viewer's ability to supports .pdfs now . . .

- [example manifest](https://iiif.wellcomecollection.org/presentation/v2/b17502792)

## Expected Behavior
- .pdfs are viewed in Universal Viewer

## Notes
- [Sherlockian periodicals (Holmes0011)](https://umedia.lib.umn.edu/search?facets%5Bparent_collection_name%5D%5B%5D=Sherlockian+periodicals+%28Holmes0011%29)
